content/tools/aws-vault.md

```diff
@@ -42,36 +42,52 @@ sudo chmod 755 /usr/local/bin/aws-vault
 ```
 
 ## Local Configuration
+We recommend using the `file` type backend for `aws-vault` because this is compatible with Linux, which is needed for [Geodesic](/geodesic) sessions.
 
-Setup your `~/.aws/config` by adding a profile entry for each AWS account.
-
-Here's an example of how we do it:
+Add the following to your `~/.bashrc`:
 
 ```
-[profile cloudposse-dev-admin]
-region=us-west-2
-role_arn=arn:aws:iam::29013231371:role/OrganizationAccountAccessRole
-mfa_serial = arn:aws:iam::313021614177:mfa/erik@cloudposse.com
-source_profile=cloudposse
+export AWS_VAULT_BACKEND="file"
 ```
 
+Then `source ~/.bashrc` to update your current session.
+
+1. Generate IAM Access Key ID/Secret on your AWS root account via IAM management page in the AWS Console.
+
 {{% dialog type="important" icon="fa fa-exclamation-triangle" title="Important" %}}
 Do not define the source profile in `~/.aws/credentials`; we're going to use `aws-vault add` for that.
 {{% /dialog %}}
 
-We recommend using the `file` type backend for `aws-vault` because this is compatible with Linux, which is needed for [Geodesic](/geodesic) sessions.
+2. Using the IAM Access Key ID/Secret generated in Step 1, add the `source_profile`:
+```bash
+$ aws-vault add example
+```
 
-Add the following to your `~/.bashrc`:
+3. Add the `source_profile` created in Step 2 to your `~/.aws/config`.
+```
+[profile example]
+region=us-west-2
+```
 
+4. Setup your `~/.aws/config` by adding a profile entry for each AWS account:
 ```
-export AWS_VAULT_BACKEND="file"
+[profile example-staging-admin]
+region=us-west-2
+role_arn=arn:aws:iam::XXXXXXXXXXXX:role/OrganizationAccountAccessRole
+mfa_serial=arn:aws:iam::XXXXXXXXXXXX:mfa/user@example.com
```

Comment:
Explain below what `XXXXXXXXXXXX` is

Reply:
Explain that `user@example.com` is their IAM username (we recommend using email addresses for all IAM user accounts associated with humans)